cheatsheet-for-tidyverse.Rmd

```diff
@@ -0,0 +1,11 @@
+# Cheat Sheet for Tidyverse
+
+Yinda Qian
+
+This is a R Cheat Sheet on Tidyverse. I made this Cheat Sheet to give RStudio user a clearer view of what Tidyverse is, how Tidyverse works, what advantages Tidyverse have, and what functions Tidyverse includes. By following each section, users will have a picture of how to utilize the function to more efficiently, manipulate data. 
+
+[Download Cheatsheet](Spring 2022/5293/Community Contribution/Cheatsheet_YindaQian_yq2324.pdf)
+
+![](Spring 2022/5293/Community Contribution/Cheatsheet_YindaQian_yq2324.pdf)
```

Comment:
the path here (and the path in line 7) is not correct. Please use relative path with respect to this repo, such as "resources/your-directory/your-file"

Reply:
Hi Yujie,
How could I change the lines? Through my Rmd. file? Thank you very much.

Reply:
> the path here (and the path in line 7) is not correct. Please use relative path with respect to this repo, such as "resources/your-directory/your-file"

@tu-yujie I just changed the path in my Rmd. file and committed again. Is this how it works? Will it be updated soon?

Reply:
Thank you! I will merge your pull request now.

Reply:
Much appreciated!